app/validators/has_one_title_validator.rb

```diff
@@ -0,0 +1,8 @@
+# validates that the title has at least one title
+class HasOneTitleValidator < ActiveModel::Validator
```

Comment:
Directly from models. It's a confusing validator, and I'm not sure why other validators (like `presence`) are insufficient.